lib/simplifyExpression/collectAndCombineSearch/divideLikeTerms.js

```diff
@@ -0,0 +1,178 @@
+const arithmeticSearch = require('../arithmeticSearch');
```

Comment:
send '/' not '-' but yes, that's what I'm thinking. You'll want to rename the file though to be more descriptive of its new purpose